realm-flipper-plugin-device/package.json

```
@@ -39,7 +39,7 @@
     "react": ">=17"
   },
   "dependencies": {
-    "react": "file:../node_modules/react",
+    "react": "latest",
```

I'd rather we pin this to a specific major version.
```suggestion
    "react": "^18.2.0",
```